app/models/answer.rb

```diff
@@ -1,3 +1,6 @@
 class Answer < ApplicationRecord
   belongs_to :question
+  validates :title, presence: true
+
+  scope :correct_anwer, ->(id) { where(question_id: id, correct: "t") }
```

Comment:
```suggestion
  scope :correct, -> { where( correct: true) }
```
1. не стоит дублировать имя класс в скоупе
2. по ТЗ нужен скоуп для выбора корректных ответов, про привязку к вопросу ничего не говорится и собственно это просто решается с помощью ассоциации
3. видимо я пропустил это - у тебя поле `correct` почему-то задано как `string`, хотя должно быть по заданию `boolean` - нужно исправить (либо откатить миграции, либо добавить миграцию и поменять тип поля), правильный синтаксис для булева поля я указал, соответсвенно дефолт нужно тоже ставить не строку, а булево (`false`)